src/Components/Home/Searchbar.js

```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { InputText } from 'primereact/inputtext'
 import GetIcons from '../Icons/GetIcons'
+import './Home.css'
```

Comment:
Is this import needed?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
Yes. That's where the CSS is for the search bar.